pkg/controller/util/node/controller_utils.go

```diff
@@ -187,6 +187,17 @@ func ExistsInCloudProvider(cloud cloudprovider.Interface, nodeName types.NodeNam
 	return true, nil
 }
 
+// ShutdownInCloudProvider returns true if the node is shutdowned in
+// cloud provider.
+func ShutdownInCloudProvider(cloud cloudprovider.Interface, node *v1.Node) (bool, error) {
+	instances, ok := cloud.Instances()
+	if !ok {
+		return false, fmt.Errorf("%v", ErrCloudInstance)
+	}
+	shutdown, err := instances.InstanceShutdownByProviderID(context.TODO(), node.Spec.ProviderID)
```

Comment:
I think you should add the not implemented check here and return `false, nil` if it's not implemented to reduce the error handling in the controllers.